### PR TITLE
[tutorials] Update list tutorial

### PR DESCRIPTION
**Pull request by augustjk**
- Change duration to 20 minutes
- Fix property misname in step 4
- Change solution in step 4 to use `.forEach` to avoid infinite loop while typing

### CODE REVIEW

File: packages/lit-dev-content/samples/tutorials/working-with-lists/03/after/my-element.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import {LitElement, html} from 'lit';
 import {customElement, property, state} from 'lit/decorators.js';
+import type {TemplateResult} from 'lit';
```
Comment (AndrewJakubowicz) [inline packages/lit-dev-content/samples/tutorials/working-with-lists/03/after/my-element.ts:3]: This type import and the `const listItems: TemplateResult[] = [];` initialization need to be updated in the `before` file.

**Review by AndrewJakubowicz — APPROVED** [review]
Pre approval - one `before` file needs to be updated with the type import.

Great way to get around the infinite loop!

File: packages/lit-dev-content/samples/tutorials/working-with-lists/03/after/my-element.ts (hunk 1 above)
Comment (augustjk) [reply packages/lit-dev-content/samples/tutorials/working-with-lists/03/after/my-element.ts:3]: Oops, thanks for catching that! It's weird that prior to this TS was happily inferring the type for the array. I guess it has trouble following what happens inside `.forEach`'s callback.

**Review by e111077 — APPROVED** [review]
sad to hear about the loops. LGTM pending andrew's suggestion

**Pull request merged**